### ISSUE TITLE
Error: Recurrent type definition

### ISSUE THREAD

**Issue by redndahead** [mulesoft/api-workbench#83]
I was playing around with types and I ran into this issue. Is this not allowed?

Given:
test.raml
````
#%RAML 1.0 Library
uses:
  ld: !include test2.raml

types:
  Post:
    type: object
    properties:
      name: string
  Base:
    type: [ld.Base, Post] <--- Error shown on this line
    properties:
      id: string
````

and

test2.raml
````
#%RAML 1.0 Library
types:
  Base:
    type: object
    properties:
      "@id": string
      "@type": string
````

**Comment by petrochenko-pavel-a** [comment]
Hi @redndahead  this is clearly a bug. We will take care about it asap.

Regards,
Pavel

**Comment by rchang-ck** [comment]
bump. how's this going?

**Comment by trevorwh** [comment]
+1 - we are also experiencing this issue - any update? Many thanks!

**Comment by ddenisenko** [comment]
It will be fixed at the next release.

**Comment by ddenisenko** [comment]
Fixed in 0.8.14.
